microsoft-edge/accessibility/build/aria-and-ui-automation.md

```diff
@@ -11,9 +11,9 @@ keywords: accessibility, accessibility for developers, accessible websites, edge
 ms.custom: seodec18
 ---
 
-# ARIA and UI Automation in Microsoft Edge
+# ARIA and UI automation in Microsoft Edge
 
-The W3C defines Accessible Rich Internet Applications (ARIA) as a syntax for making dynamic web content and custom UI accessible. Microsoft Edge recognizes the ARIA role, state, and property information and exposes it to assistive technologies, which in turn can use the [Microsoft UI Automation](https://blogs.msdn.microsoft.com/winuiautomation/) APIs to retrieve the information.
+The W3C defines Accessible Rich Internet Applications (ARIA) as a syntax for making dynamic web content and custom UI accessible. Microsoft Edge recognizes the ARIA role, state, and property information and exposes it to assistive technologies, which in turn can use the [Microsoft UI Automation](https://blogs.msdn.microsoft.com/winuiautomation/) APIs to retrieve the information. 
 
 Visit [HTML5Accessibility](https://html5accessibility.com) for information on which new HTML5 features are accessibly supported by Microsoft Edge.
 
```

Comment:
I thought the "Microsoft Edge rendering engine (EdgeHTML)" was the "old way" ... compared to the new Chromium based version

Reply:
Limitations of GitHub PR UI: it only shows what it things are the changed lines.   Line 20 which you commented about is not shown, so it's hard to see what you're referring to.   Could you please click + on line 20 and re-enter your PR review comment there?  @vchapel

Reply:
I can't get a plus sign at unchanged lines ... is there a trick to get this?

Reply:
That's something to keep on our list of investigations of the GitHub PR UI.  I have figured out several valuable things about this GitHub PR Review UI, so that is worth pursuing.

Reply:
> Limitations of GitHub PR UI: it only shows what it thinks are the "changed lines". Line 20, which you commented about, is not shown, so it's hard to see what you're referring to. Could you please click + on line 20 and re-enter your PR review comment there? @vchapel
> Looking again at this problem, maybe there's no way to do that. ... You could quote the line in your comment, that would help.
> Line 20:
> The Microsoft Edge rendering engine (EdgeHTML) builds an accessible projection of web pages, conforming to the following W3C specifications:
> I plan to delete " (EdgeHTML)" if that would work.

Awesome Blossom